app/Main.hs

```diff
@@ -252,19 +281,33 @@ main = do
   localPath <- maybe (getEnv "HOME") return oLocalPath
   eS3Bucket <- lookupEnv "S3_BUCKET"
   let gitAccount = "lehins" -- "commercialhaskell"
-      delay = fromMaybe 60 oDelay * 100000
+      delay = fromMaybe 60 oDelay * 1000000
       ms3Bucket = msum [oS3Bucket, eS3Bucket]
       gitUser =
         GitUser
         { userName = oUsername
         , userEmail = oEmail
         , userGPG = oGPGSign
         }
+  case (ms3Bucket, oAwsAccessKey, oAwsSecretKey) of
+    (Just _, Just accessKey, Nothing) -> setEnv "AWS_ACCESS_KEY_ID" accessKey
```

Comment:
If either `aws-access-key` or `aws-secret-key` is set, but not both, it will set proper variables and using `Discover` should pick them up. This way if one is passed as an argument and another through an environment variable, it should still work. Do you think it is better just to throw an error if either one is missing?